Pipfile

```diff
@@ -16,4 +16,4 @@ django-cors-headers = "*"
 [dev-packages]
 
 [requires]
-python_version = "3.13"
+python_version = "3.12"
```

Comment:
this is not needed. THe problem is not happening due to the python version, but due to the database. Django tries to connect to the PostgresDB before it starts up, if it can't the static files (html, css) don't load